src/components/ebay-breadcrumb/test/test.browser.js

```diff
@@ -0,0 +1,47 @@
+const sinon = require('sinon');
+const expect = require('chai').expect;
+const testUtils = require('../../../common/test-utils/browser');
+const mock = require('../mock');
+const renderer = require('../');
+
+describe('breadcrumb with prevent-default state', () => {
+    let widget;
+    let list;
+    describe('when breadcrumb item is clicked', () => {
+        let clickSpy;
+        beforeEach((done) => {
+            widget = renderer.renderSync(mock.itemsWithPreventDefault).appendTo(document.body).getWidget();
```

Comment:
Rendering should be done in the outermost `describe()` -- check some of the existing tests for this setup.

Reply:
it has two set of inputs `mock.itemsWithPreventDefault` and `mock.basicItems` to render. i do not see value of moving out to describe.

Reply:
You're right that it doesn't make a practical difference for very simple cases like this, but this is part of the structure we've been following in browser tests. The outermost `describe()` (which should start with "given ...") describes a given initial setup, so rendering fits naturally in that stage.